pkg/cmd/attestation/verification/extensions.go

```diff
@@ -31,51 +31,31 @@ func VerifyCertExtensions(results []*AttestationProcessingResult, tenant, owner,
 	}
 }
 
-func verifyCertExtensions(attestation *AttestationProcessingResult, tenant, owner, repo, issuer string) error {
-	var want string
-
-	if tenant == "" {
-		want = fmt.Sprintf("https://github.com/%s", owner)
-	} else {
-		want = fmt.Sprintf("https://%s.ghe.com/%s", tenant, owner)
-	}
-	sourceRepositoryOwnerURI := attestation.VerificationResult.Signature.Certificate.Extensions.SourceRepositoryOwnerURI
-	if !strings.EqualFold(want, sourceRepositoryOwnerURI) {
-		return fmt.Errorf("expected SourceRepositoryOwnerURI to be %s, got %s", want, sourceRepositoryOwnerURI)
+func verifyCertExtensions(attestation *AttestationProcessingResult, c EnforcementCriteria) error {
```

Comment:
i know i said let's keep this PR short and sweet but since we just need the `Signature.Certificate` shall we shorten this up by changing the func sig to:

```suggestion
func verifyCertExtensions(verifiedCert certificate.Summary, criteria EnforcementCriteria) error {
```